### PR TITLE
I'm being forced to update

### PR DESCRIPTION
**Pull request by ViggoRu21**

### CODE REVIEW

File: applicant/templates/applicant/listing_detail.html [hunk 1]
```diff
@@ -6,28 +6,35 @@
...
+    <div>
+    
+        <a class="or_button" href="{% url 'applicant:choose_info' uid=1 lid=1%}">
+            Apply
+        </a>
+    </div>
 
+    <div>
+        <h2>Job Description</h2>
+        <h3>aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa</h3>
```
Comment (evaxrut) [inline applicant/templates/applicant/listing_detail.html:37]: nei

File: applicant/templates/applicant/login.html [hunk 2]
```diff
@@ -30,19 +32,17 @@ <h2>Log in to Your Account</h2>
         {% endif %}
     </div>
 
-    
     <div class="divider">
         <h3>
             \
```
Comment (evaxrut) [inline applicant/templates/applicant/login.html:37]: hafa 'or' frekar ???

File: applicant/templates/applicant/listing_detail.html (hunk 1 above)
Comment (ViggoRu21) [reply applicant/templates/applicant/listing_detail.html:37]: im literally working on this page rn
u interrupted me to **FORCE** me to update

Comment (evaxrut) [reply applicant/templates/applicant/listing_detail.html:37]: fine

Comment (evaxrut) [reply applicant/templates/applicant/listing_detail.html:37]: for now...

Comment (evaxrut) [reply applicant/templates/applicant/listing_detail.html:37]: ekki gleyma að taka þetta

**Review by evaxrut — APPROVED** [review]

File: applicant/templates/applicant/login.html (hunk 2 above)
Comment (ViggoRu21) [reply applicant/templates/applicant/login.html:37]: or is uglier but we could change it back
breytti úr or, gætum lika fundið eh
          /
       or
      /
svg image

**Review by MFernirjons — APPROVED** [review]

**Pull request merged**